www/src/pages/getting-started.md

```diff
@@ -3,6 +3,7 @@
 
 ```sh
 npm i rebass
+yarn add rebass
```

Comment:
Thanks, but not sure I understand the intent here. This would be redundant if you've already installed with `npm i` – Rebass doesn't require the use of yarn, and I suspect most yarn users would know how to swap out `npm` CLI commands for yarn ones. I'd like to keep this as beginner-friendly as possible

Reply:
Okay, that's fine 👍